src/scanoss/cli.py

```
@@ -1356,13 +1389,73 @@ def inspect_undeclared(parser, args):
+               command line parser object
+           args: Namespace
+               Parsed arguments
+       """
+    if args.input is None:
+        print_stderr('Please specify an input file to inspect')
+        parser.parse_args([args.subparser, args.subparsercmd, '-h'])
+        sys.exit(1)
+    output: str = None
+    if args.output:
+        output = args.output
+        open(output, 'w').close()
```

Pylint (3.3.7)

[refactor] Consider using 'with' for resource-allocating operations

(R1732)